### PR TITLE
Generate `any` in case of error

### PR DESCRIPTION
**Pull request by vvscode**
Closes https://github.com/KuveytTurk/react-dts-generator/issues/2

### CODE REVIEW

File: test/anyFallback.test.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { assert } from 'chai';
+import { generate } from '../src';
+
+describe('ArrayOf test', () => {
```
Comment (mozkarakoc) [inline test/anyFallback.test.ts]: could you provide a clearer test name?

Comment (vvscode) [inline test/anyFallback.test.ts]:
```suggestion
describe('anyFallback test', () => {
```

Comment (vvscode) [reply test/anyFallback.test.ts]: Done, sorry

**Comment by vvscode** [comment]
Sorry, didn't get about the typo in baselines

File: baselines/anyFallback/anyFallback.d.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+
+export interface AnyFabllbackComponentProps {
```
Comment (mozkarakoc) [inline baselines/anyFallback/anyFallback.d.ts]: AnyFabllbackComponent has a typo

Comment (vvscode) [reply baselines/anyFallback/anyFallback.d.ts]: Fixed

**Pull request merged**